tests/utils.go

```diff
@@ -154,3 +155,24 @@ func MountCloudInitFunc(devName string) func(*v1.VirtualMachineInstance) {
 		Expect(err).ToNot(HaveOccurred())
 	}
 }
+
+func DisableDownwardMetrics(client kubecli.KubevirtClient) {
+	kv := libkubevirt.GetCurrentKv(client)
+	kv.Spec.DownwardMetrics = nil
+
+	updateKubevirtSpec(kv)
+}
+
+func EnableDownwardMetrics(client kubecli.KubevirtClient) {
+	kv := libkubevirt.GetCurrentKv(client)
+	kv.Spec.DownwardMetrics = &v1.DownwardMetricsConfiguration{}
+
+	updateKubevirtSpec(kv)
```

Comment:
Use `libkubevirt.UpdateKubeVirtConfigValueAndWait` and drop `updateKubevirtSpec`.

Reply:
As far as I was able to understand, `UpdateKubeVirtConfigValueAndWait` will just update the `kv.Spec.Configuration`, while the downwardMetrics configurable is under `kv.Spec`. Therefore, if I use `UpdateKubeVirtConfigValueAndWait` the change won't be reflected.

Reply:
I see, can `libkubevirt` be updated to do what we want?

Reply:
We are updating the `kv.Spec`, that's what `updateKubevirtSpec` is already doing. Sorry, maybe I'm missing something here. Could you please elaborate a bit more?

Reply:
Still, can libkubevirt be used instead? And the Disable/Enable helpers should live closer to where they are used. We should not grow `tests/utils.go` at all.